Marlin/src/config/examples/wt150/Configuration.h

```diff
@@ -1108,7 +1108,10 @@
 
   // Probe along the Y axis, advancing X after each column
   //#define PROBE_Y_FIRST
-
+  
+  // Enable G29 G parameter to set dimension of square centered probing area
+  //#define G29_SQUARE_GRID_SIZE
+  
```

Comment:
Please strip all trailing whitespace from these new lines, or better yet, just drop this extra configuration option altogether.

Reply:
@thinkyhead     I believe this only affects:
```cpp
#if ENABLED(AUTO_BED_LEVELING_LINEAR) || ENABLED(AUTO_BED_LEVELING_BILINEAR)
```

Reply:
Ah, quite so!